MAUI/Popup/localization.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Localization in .NET MAUI SfPopup control | Syncfusion
```

Comment:
Title: Localization in .NET MAUI SfPopup control | Syncfusion

Reply:
not changed